integration/dbt/scripts/dbt-ol

```diff
@@ -14,13 +15,29 @@ from openlineage.common.utils import parse_single_arg
 PRODUCER = f'https://github.com/OpenLineage/OpenLineage/tree/{__version__}/integration/dbt'
 
 
+def setup_client() -> Optional[OpenLineageClient]:
+    url = os.getenv("OPENLINEAGE_URL")
+    if not url:
+        print("OPENLINEAGE_URL is not set: stopping execution")
+        return None
+    return OpenLineageClient(
+        url=url,
+        options=OpenLineageClientOptions(
+            api_key=os.getenv("OPENLINEAGE_API_KEY", None)
+        )
+    )
+
+
 def main():
     args = sys.argv[2:]
     target = parse_single_arg(args, ['-t', '--target'])
     project_dir = parse_single_arg(args, ['--project-dir'], default='./')
     profile_name = parse_single_arg(args, ['--profile'])
 
-    client = OpenLineageClient.from_environment()
+    client = setup_client()
+    if client is None:
+        sys.exit(1)
```

Comment:
for readability, I would collocate the check the message and the sys.exit

Reply:
Done.